package.json

```diff
@@ -10,11 +10,17 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.5.2",
+    "@fortawesome/free-solid-svg-icons": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.2",
     "bootstrap": "^5.3.3",
+    "bootstrap-icons": "^1.11.3",
     "react": "^18.2.0",
     "react-bootstrap": "^2.10.2",
+    "react-bootstrap-sidebar-menu": "^2.0.3",
```

Comment:
I would avoid adding this external library. We want less dependencies on third-party stuff.
It is ok to use some famous third-party libraries like `fortawesome`, `bootstrap` etc.

This one is ok for now. Just something to consider for future.